config/webpack-metalsmith-connect/metalsmith-webpack.js

```diff
@@ -88,7 +89,11 @@ function webpackDevServerPlugin(config, opts) {
     server.listen(options.port || 8081, options.host, () => {
       console.log(
         chalk.blue('[metalsmith-webpack-dev-server]: ') +
-        chalk.green(`Running webpack dev server at http://${options.host}:${options.port}`)
+          chalk.green(
+            `Running webpack dev server at http://${options.host}:${
+              options.port
+            }`,
```

Comment:
I'm not a fan of this, though. Why did this get split to new lines? Line length limit?